### ISSUE TITLE
new parameter: playback start time

### ISSUE THREAD

**Issue by jweu** [huceke/omxplayer#194]
Is it possible to specify the display time of the first frame in milliseconds ? This would be nice to synchronize multple players for an art video installation.

**Comment by popcornmix** [comment]
I don't really understand. What would be the time you specified?
Absolute time of day (e.g. 14:23:41.123) ?
This would require you had good clock sync (e.g. ntp) of the Pi's.
Even with this, the clocks would still drift whilst the video was playing.

**Comment by jweu** [comment]
Yes, lets use the absolute time like 14:23:41.123 . Of course we need ntp and a server with the next expected start time.

Why should the video drift if ntp is synchronizing the clocks? The video-clock is not the system clock and even unstable? How stable are these clocks? I hope, adding the start time is not a big change to omxplayer but would help a lot in this situation. Most art videos are several minutes long.

**Comment by popcornmix** [comment]
The timing of video is determined by the 19.2MHz oscillator (which determines the audio clock).
This will be something like 100ppm.
So, after a 1hour video, it may have drifted by 1/3 of a second.

**Comment by jweu** [comment]
Is the drift constant for each raspberry? If so, one could sell the slowest and fastest at ebay and buy new ones ;-) For some videos a drift by 1/3 of a second per hour is ok, its one frame at 6 minutes.

**Comment by popcornmix** [comment]
The drift will not be constant. Temperature will affect it.

**Comment by jweu** [comment]
Thanks for your remarks. If they are at the same room, the temperature should be the same, lets try multiple screen installations.

**Comment by DoodleIncident** [comment]
Have you tried just setting up a cron job or something to tell omxplayer to unpause? You can control it by telling it to listen to a FIFO for its input, and then just echo things into that pipe when you want it to unpause. http://www.raspberrypi.org/forums/viewtopic.php?f=38&t=7987
